app/views/recipes/show.html.erb

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Public Recipes</title>
+    <%= csrf_meta_tags %>
+    <%= action_cable_meta_tag %>
+    <%= stylesheet_link_tag "application",
+    media: "all",
+    "data-turbolinks-track": "reload" %>
+    <%= javascript_include_tag "application", "data-turbolinks-track": "reload" %>
+  </head>
+  <body>
+
+    <%= render "shared/nav", user: @user %>
+    <div class="display-recipe">
+      <h2 class="recipe-title"><%= @recipe.name %></h2>
+      <div class="recipe-details">
+
+        <div class="recipe-info">
+          <p class="recipe-specs">Preparation time:
+            <%= @recipe.preparation_time %>
+            min(s)</p>
+        </div>
+
+        <p class="recipe-specs">Cooking time:
+          <%= @recipe.cooking_time %>
+          min(s)</p>
+      </div>
+
+      <div class="public">
+        <% if @recipe.user_id == @user.id %>
+          <%= form_for @recipe, url: recipe_path(@recipe), method: :patch, class: "form-public" do |f| %>
+            <%= f.label :public, "Public", class: "label-public" %>
+            <%= f.check_box :public,
+                        class: "public-checkbox",
+                        checked: @recipe.public,
+                        onclick: "this.form.submit();" %>
+          <% end %>
+        <% end %>
+      </div>
```

Comment:
- [ ] It would be nice if you match the given wireframe with how your page looks
your design :point_down: 
![Screenshot from 2023-06-29 11-59-32](https://github.com/Ade179/wine-and-dine/assets/66097371/2025bfe5-070b-4002-8eb8-0fe26a251d75)
wireframe :point_down: 
![Screenshot from 2023-06-29 12-00-33](https://github.com/Ade179/wine-and-dine/assets/66097371/601cc38c-7eb5-446f-a5a3-c6701d9700f5)